src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def user_filesystem(tmp_path):
+    directory = Path(tmp_path)
+    os.chdir(directory)
+
+    input_dir = Path(tmp_path).resolve() / "input_dir"
+    input_dir.mkdir(parents=True, exist_ok=True)
+
+    chi_data = "dataformat = twotheta\n mode = xray\n # chi_Q chi_I\n 1 2\n 3 4\n 5 6\n 7 8\n"
+    xy_data = "1 2\n 3 4\n 5 6\n 7 8"
+    txt_data = "dataformat = twotheta\n mode = xray\n # chi_Q chi_I\n 1 2\n 3 4\n 5 6\n 7 8\n"
```

Comment:
what is your intent with this txt_data?  It seems to be t he same as a chi file but with a different name, so what do you want to test with this?  I can think of some things but I was wondering your intent.